sparse/src/KokkosSparse_BsrMatrix.hpp

```diff
@@ -872,13 +876,13 @@ class BsrMatrix {
 
   //! The number of "point" (non-block) rows in the matrix.
   //  This is the dimension of the range of this matrix as a linear operator.
-  KOKKOS_INLINE_FUNCTION ordinal_type numPointRows() const {
+  KOKKOS_INLINE_FUNCTION ordinal_type numPointVectors() const {
```

Comment:
Same as below why Vectors/ Elems?

Reply:
This is an artifact of refactoring CrsMatrix to the row/column agnostic parent class CompressedSparseMatrix. Both CrsMatrix member and BsrMatrix members are used interchangeably in sparse kernels. I renamed these to match the members of CompressedSparseMatrix -- in the long run, I would advocate for refactoring BsrMatrix as well. For now, I will add `numPointRows` and `numPointCols` wrappers to CrsMatrix

Reply:
Also, it looks like numPoint* can be removed in favor of numRows() and numCols().